### PR TITLE
Added traffic_model parameter to directions requests

### PR DESCRIPTION
**Pull request by markmcd**
PTAL @domesticmouse @samthor

### CODE REVIEW

File: src/main/java/com/google/maps/DirectionsApiRequest.java [hunk 1]
```diff
@@ -133,7 +138,8 @@ public DirectionsApiRequest arrivalTime(ReadableInstant time) {
   }
 
   /**
-   * Set the departure time for a Transit directions request. If not provided, "now" is assumed.
+   * Set the departure time for a transit or driving directions request. If not provided, "now" is
```
Comment (samthor) [inline src/main/java/com/google/maps/DirectionsApiRequest.java]: This seems incongruous with the comment on 57? That you need to specify departure_time?

File: src/main/java/com/google/maps/model/TrafficModel.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package com.google.maps.model;
```
Comment (samthor) [inline src/main/java/com/google/maps/model/TrafficModel.java]: copyright headers

Comment (markmcd) [inline src/main/java/com/google/maps/model/TrafficModel.java]: done

**Comment by domesticmouse** [comment]
LGTM

**Comment by markmcd** [comment]
Thanks team - I don't want to merge this just yet, so leave it to me to do please.

**Pull request merged**